pkg/invariants/disruptionimageregistry/invariant.go

```diff
@@ -47,6 +50,19 @@ func NewRecordAvailabilityOnly() invariants.InvariantTest {
 func (w *availability) StartCollection(ctx context.Context, adminRESTConfig *rest.Config, recorder monitorapi.RecorderWriter) error {
 	var err error
 
+	coreClient, err := e2e.LoadClientset(true)
+	if err != nil {
+		return fmt.Errorf("unable to load client set: %v", err)
+	}
+	isMicroShift, err := exutil.IsMicroShiftCluster(coreClient)
+	if err != nil {
+		return fmt.Errorf("unable to determine if cluster is MicroShift: %v", err)
+	}
+	if isMicroShift {
+		w.notSupportedReason = "Platform MicroShift not supported"
```

Comment:
follow the pattern of the other by checking the len of notSupportedReason below this for easy of future expansion and you'll want the other steps to also skip on the same len check